Add OS-Agent unsupported message (#19427)
home-assistant/home-assistant.github.io: source/more-info/unsupported/os_agent.markdown

---
title: "OS-Agent issues"
description: "More information on why missing OS-Agent marks the installation as unsupported."
---

## The issue

> OS-Agent is how the Supervisor does handle additional tasks on the host,
> without this, multiple things that the Supervisor needs to do will fail.

Reviewer: ```suggestion
OS-Agent is how the Supervisor handles additional tasks on the host,
without it, the Supervisor won't be able to fulfill their tasks and responsibilities.
```

> without it, the Supervisor won't be able to fulfill their tasks and responsibilities.

Reviewer 2: ```suggestion
without it, the Supervisor won't be able to fulfill its tasks and responsibilities.
```

Reviewer 2: nah...

## The solution

> Home Assistant Operating-System have this preinstall since Vesion 6.0 - you need maybe reboot your system.

Reviewer: ```suggestion
Home Assistant Operating-System has had this preinstalled since version 6.0 - you need maybe reboot your system.
```
